### PR TITLE
[Doc]--Add example of offloading data to NFS

### PR DESCRIPTION
**Pull request by Huanli-Meng**
Update Tiered Storage > Filesystem offloader document

- Contents update:

  1. Add example of offloading data to NFS
  2. Refine the doc structure of the existing contents.

- Affected releases:

  1. Since Pulsar 2.7.0 

**Note**: content updates are same for all releases. You can just review doc update in one release and I will update your comments to other releases together.
  
- [x ] `doc` 
  
  (If this PR contains doc changes)

### CODE REVIEW

File: site2/docs/tiered-storage-filesystem.md [hunk 1]
```diff
@@ -265,4 +204,312 @@ To trigger via CLI tools, you need to specify the maximum amount of data (thresh
 
 ## Tutorial
 
-For the complete and step-by-step instructions on how to use the filesystem offloader with Pulsar, see [here](https://hub.streamnative.io/offloaders/filesystem/2.5.1).
\ No newline at end of file
+This section provides step-by-step instructions on how to use the filesystem offloader to move data from Pulsar to Hadoop Distributed File System (HDFS) or Network File system (NFS).
+
+<!--DOCUSAURUS_CODE_TABS-->
```
Comment (Anonymitaet) [inline site2/docs/tiered-storage-filesystem.md:209]: Have you previewed the content using `yarn start`?

File: site2/docs/tiered-storage-filesystem.md [hunk 2]
```diff
@@ -265,4 +204,312 @@ To trigger via CLI tools, you need to specify the maximum amount of data (thresh
...
+
+To move data from Pulsar to HDFS, follow these steps.
+
+### Step1: Prepare the HDFS environment
+
+This tutorial sets up a Hadoop single node cluster and uses Hadoop 3.2.1.
+
+> **Tip**
+>
+> For more information about setting up a Hadoop single node cluster, see [here](https://hadoop.apache.org/docs/stable/hadoop-project-dist/hadoop-common/SingleCluster.html).
```
Comment (Anonymitaet) [inline site2/docs/tiered-storage-filesystem.md:221]:
```suggestion
> For how to set up a Hadoop single node cluster, see [here](https://hadoop.apache.org/docs/stable/hadoop-project-dist/hadoop-common/SingleCluster.html).
```

How about changing the description more concise and direct?

File: site2/docs/tiered-storage-filesystem.md [hunk 3]
```diff
@@ -265,4 +204,312 @@ To trigger via CLI tools, you need to specify the maximum amount of data (thresh
...
\ No newline at end of file
+This section provides step-by-step instructions on how to use the filesystem offloader to move data from Pulsar to Hadoop Distributed File System (HDFS) or Network File system (NFS).
+
+<!--DOCUSAURUS_CODE_TABS-->
+
+<!--HDFS-->
+
+To move data from Pulsar to HDFS, follow these steps.
+
+### Step1: Prepare the HDFS environment
```
Comment (Anonymitaet) [inline site2/docs/tiered-storage-filesystem.md:215]:
```suggestion
### Step 1: Prepare the HDFS environment
```

File: site2/website-next/docs/tiered-storage-filesystem.md [hunk 4]
```diff
@@ -12,307 +12,526 @@ This chapter guides you through every step of installing and configuring the fil
...
 ### Configure filesystem offloader driver
 
-You can configure filesystem offloader driver in the configuration file `broker.conf` or `standalone.conf`.
+You can configure the filesystem offloader driver in the `broker.conf` or `standalone.conf` configuration file.
 
 - **Required** configurations are as below.
+
+    <!--DOCUSAURUS_CODE_TABS-->
+
+    <!--HDFS-->
```
Comment (Anonymitaet) [inline site2/website-next/docs/tiered-storage-filesystem.md:92]: When adding multiple tabs for docs in `website-next`, please adhere to the new syntax: https://docs.google.com/document/d/1IV35SI_F8G8cL-Vuzknc6RTGLK9_edRMpZpnrHvAWNs/edit#bookmark=id.xmxlbpg3yug1

You can take this (https://github.com/apache/pulsar/blob/master/site2/website-next/versioned_docs/version-2.7.1/admin-api-namespaces.md) as a reference

**Comment by Huanli-Meng** [comment]
@Anonymitaet docs are updated and PTAL again. And i met error when running the `yarn start `command. I tried to upgrade the nodejs. but still fails.

**Review by Anonymitaet — APPROVED** [review]

**Pull request merged**